### DISCUSSION TITLE
I have LightSpeed Filter agent and it isn't showing up when I us the code

### DISCUSSION THREAD

**Discussion by BreathoftheWildLink** [3kh0/ext-remover#303] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
It shows these two extensions but not the blocking extension LightSpeed
![Capture](https://user-images.githubusercontent.com/119606384/205940382-c71bcc85-a47f-422c-9bdf-868df2b7c269.PNG)

**Comment by Aka-but-nice** [comment]
Is Lightspeed an app?

**Reply by BreathoftheWildLink** [reply]
I guess it is, that's sad, (+1 for schools)

**Comment by nicolasog12** [comment, marked as answer]
LSRelay should be an extension.
Go to your extensions page (chrome://extensions or your browser extensions page) and look for Lightspeed Filter Agent
![image](https://user-images.githubusercontent.com/99337184/205947559-075881c0-bdcf-489d-abff-2c9bb897e2bb.png)
If you don't see this, your school is not using the LSRelay extension, and using another program.
(_make sure you are using chrome and any version below v106!!!_) IF you use v106 or above, it won't work.

**Reply by BreathoftheWildLink** [reply]
Thx, I guess that was it, I have to rely on proxies now :(

**Reply by brebathe** [reply]
how do I do this, my school uses LSRelay

**Comment by rwg100** [comment]
you can right click on the extension in the top right corner
click on the extensions name
ex: dyknow cloud
it should bring you to the chrome page

at the top the ID should be in the URL
